packages/iris-grid/src/IrisGridTableModel.js

```diff
@@ -1044,8 +1072,17 @@ class IrisGridTableModel extends IrisGridModel {
     }
   );
 
-  isColumnMovable(x) {
-    return !this.isKeyColumn(x);
+  isColumnMovable(column) {
+    const columnName = Number.isInteger(column)
```

Comment:
This should always be the columnIndex (a `number`) passed in, not the column name. Then with the column index, retrieve the column with just `this.columns[column].name`

This method is defined in `GridModel`, and `Grid` itself has no concept of columns with names, it just references columns by numbers for performance. Moved columns are applied as a transformation.